src/components/Counter/Counter.tsx

```diff
@@ -92,7 +102,7 @@ export class Counter extends React.Component<CounterProps> {
           name="increment"
           disabled={disabled || (max && value + step > max)}
         >
-          <Plus />
+          <Plus className={sizeClass} />
```

Comment:
It's better to avoid using the same class for different elements, since it's harder to target them and make changes.

Instead of passing the `sizeClass` to different elements, you should just pass it to the root element and target the other elements with it:
```css
.xSmall .btnIcon { ... }
.xSmall .error { ... }
```
etc.